doc/guides/west/manifest.rst

```diff
@@ -1179,6 +1189,428 @@ processed in this order:
+
+In this case, ``groupA`` is allowed: the ``manifest.groups`` configuration
+option has higher precedence than the ``manifest: groups: [-groupA]`` content
+in the manifest file.
+
+Therefore, projects ``foo`` and ``bar`` are both active.
+
+Example 6: Overriding multiple blocked groups via configuration
+---------------------------------------------------------------
+
+The entire manifest file is:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: foo
+       - name: bar
+         groups:
+           - groupA
+       - name: baz
+         groups:
+           - groupA
+           - groupB
+
+     groups: [-groupA,-groupB]
+
+     defaults:
+       remote: example-remote
+     remotes:
+       - name: example-remote
+         url-base: https://git.example.com
+
+The ``manifest.groups`` configuration option is set to ``groupA,groupB``
+(you can ensure this by running ``west config manifest.groups
+"groupA,groupB"``).
+
+In this case, both ``groupA`` and ``groupB`` are allowed, because the
+configuration value overrides the manifest file for both groups.
+
+Therefore, projects ``foo`` and ``bar`` are both active.
+
+Example 7: Blocking multiple groups via configuration
+-----------------------------------------------------
+
+The entire manifest file is:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: foo
+       - name: bar
+         groups:
+           - groupA
+       - name: baz
+         groups:
+           - groupA
+           - groupB
+
+     defaults:
+       remote: example-remote
+     remotes:
+       - name: example-remote
+         url-base: https://git.example.com
+
+The ``manifest.groups`` configuration option is set to ``-groupA,-groupB``
+(you can ensure this by running ``west config manifest.groups --
+"-groupA,-groupB"``).
+
+In this case, both ``groupA`` and ``groupB`` are blocked.
+
+Therefore, projects ``foo`` and ``bar`` are both inactive.
+
+.. _west-manifest-submodules:
+
+Git Submodules in Projects
+**************************
+
+You can use the ``submodules`` keys briefly described :ref:`above
+<west-manifest-files>` to force ``west update`` to also handle any `Git
+submodules`_ configured in project's git repository. The ``submodules`` key can
+appear inside ``projects``, like this:
+
+.. code-block:: YAML
+
+   manifest:
+     projects:
+       - name: some-project
+         submodules: ...
+
+The ``submodules`` key can be a boolean or a list of mappings. We'll describe
+these in order.
+
+Option 1: Boolean
+=================
+
+This is the easiest way to use ``submodules``.
+
+If ``submodules`` is ``true`` as a ``projects`` attribute, ``west update`` will
+update the project's Git submodules whenever it updates the project itself. If
+it's ``false`` or missing, it has no effect.
+
+For example, let's say you have a source code repository ``foo``, which has
+some submodules, and you want ``west update`` to keep them in sync, along with
```

Comment:
It's quite understandable that this way is used for updating all submodules, but I would just emphasize it by: "... and you want ``west update`` to keep all of them in sync ...".